logger_manager.py

```diff
@@ -0,0 +1,45 @@
+# Standard library imports
+import logging
+import logging.handlers
+import os
+
+
+def setup_logging(log_name, log_level=logging.INFO):
```

Comment:
@bhuvanaurora I'm not sure if this has been discussed already, but it might be nice if these logs could be written in ECS format, so that a Filebeat instance could easily ship them to the same logs index pattern that our "normal" connector logs are in.

Reply:
Good point. I will bring it up in the discussion.

Reply:
Done